ui/app/src/models/Item.ts

```diff
@@ -107,3 +109,8 @@ export interface PasteItem {
   path: string;
   children?: PasteItem[];
 }
+
+export interface ItemWithChildren {
```

Comment:
==> `GetItemWithChildrenResponse`. Move to it's own file like GetChildrenResponse